class.jetpack-gutenberg.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/**
+ * General Gutenberg editor specific functionality
+ */
+class Jetpack_Gutenberg {
+	/**
+	 * Check if Gutenberg editor is available
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return bool
+	 */
+	public static function is_gutenberg_available() {
+		return function_exists( 'register_block_type' );
+	}
```

Comment:
Note to follow and rebase with this PR in case it gets merged first: https://github.com/Automattic/jetpack/pull/10132#discussion_r226252748